src/features/lib/createFieldMappings.ts

```
@@ -0,0 +1,19 @@
+export const createFieldMappings = (response: Record<string, any>[]) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)